css/style.css

```diff
@@ -302,3 +302,211 @@ body {
 #last-hr-line > img {
   width: 100%;
 }
+
+.contact-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-top: 134px;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.contact-container > .contact-intro {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.contact-intro > .contact-header {
+  display: flex;
+  justify-content: center;
+}
+
+.contact-header > h1 {
+  text-align: center;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  color: rgb(147, 144, 255);
+}
+
+.contact-intro > .contact-desc {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.contact-desc > p {
+  display: flex;
+  justify-content: center;
+  font-family: 'Inter', sans-serif;
+  text-align: center;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  text-indent: 1.5cm;
+  color: rgb(245, 245, 255);
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+}
+
+.contact-inbox {
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+  background-color: white;
+  margin-bottom: 24px;
+}
+
+.contact-inbox > input[type="text"] {
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: rgb(129, 129, 140);
+}
+
+.contact-inbox > input[type="text"]:valid {
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: rgb(129, 129, 140);
+}
+
+.contact-inbox > input[type="text"]:invalid {
+  border: 4px dashed red;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: rgb(129, 129, 140);
+}
```

Comment:
- [ ] Please make sure to follow the correct Figma template. Currently, when the page is loaded, the user is greeted with a form where all the forms have a dotted border. 
- [ ] The `line-indent` property of the text makes it stick to the right side currently, please make sure it is centered, as per template requirement.

![](https://i.imgur.com/LLjCqaN.png)

Reply:
@ReshyResh Noted.
I hope now I fix above question